application_store_api/namespace_fund/data_store/store.py

```diff
@@ -0,0 +1,117 @@
+import datetime
+import json
+import uuid
+
+from application_store_api.namespace_fund.data_store.data import (
+    funding_service_design_application,
+)
+from application_store_api.namespace_fund.data_store.data import (
+    initial_fund_store_state,
+)
+from dateutil import parser as date_parser
+from dateutil.tz import UTC
+from slugify import slugify
+
+"""
+Data Access Object
+A data access object (DAO) is a pattern that provides an abstract interface
+to some type of database or other persistence mechanism.
+"""
+
+
+class application_data_access_object(object):
+    def __init__(self):
```

Comment:
Classes should conventionally be CamelCase rather than snake_case right?

Reply:
This is [correct](https://www.python.org/dev/peps/pep-0008/#class-names)